README.md

```diff
@@ -125,28 +125,48 @@ which destroys the input `A` but also does not need to allocate as much memory.
 
 These use a very similar syntax to BSplines, with the major exception
 being that one does not get to choose the grid representation (they
-are all `OnGrid`).
+are all `OnGrid`). As such one must specify a set of coordinate arrays
+defining the knots of the array.
+
+In 1D
 ```jl
-A = rand(8,20)
-itp = interpolate(A, Gridded{Linear})
-itp[4,1.2] 
+A = rand(20)
+A_x = collect(1.:40.)
+knots = (a,)
+itp = interpolate(knots,A, Tuple{Gridded{Linear}})
+itp[2.0] 
 ```
 
-If the spacing between adjacent samples differs, then you can use the syntax
+The spacing between adjacent samples need not be constant, you can use the syntax
 ```jl
 itp = interpolate(knots, A, options...)
 ```
-where `knots = (xknots, yknots, ...)` specifies the positions along
-each axis at which the array `A` is sampled.
+where `knots = (xknots, yknots, ...)` to specify the positions along
+each axis at which the array `A` is sampled for arbitrary ("rectangular") samplings.
 
 For example:
 ```jl
 A = rand(8,20)
 knots = ([x^2 for x = 1:8], [0.2y for y = 1:20])
-itp = interpolate(knots, A, Gridded{Linear})
+itp = interpolate(knots, A, Tuple{Gridded{Linear},Gridded{Linear}})
```

Comment:
This tuple shouln't be needed either; does it not work for you with just `Gridded{Linear}`?